input/fsh/SD_Procedures.fsh

```diff
@@ -1,79 +1,155 @@
 * recorder only Reference(Practitioner or PractitionerRole)
 * performer.actor only Reference(Practitioner or PractitionerRole or Organization)
+* bodySite from RadiationTargetBodySiteVS (extensible)
 * bodySite.extension contains
     LocationQualifier named locationQualifier 0..1 MS
-* reasonCode and reasonReference and bodySite MS
-
+* performed[x] only Period
 
-Profile:  CancerRelatedRadiationProcedure
+Profile:  RadiotherapyCourseSummary
 Parent:   USCoreProcedure
-Id:       mcode-cancer-related-radiation-procedure
-Title:    "Cancer-Related Radiation Procedure"
-Description: "A radiological treatment addressing a cancer condition. The scope of this profile has been narrowed to cancer-related procedures by constraining the reasonReference and reasonCode to cancer conditions, one of which is required.
+Id:       radiotherapy-course-summary
+Title:    "Radiotherapy Course Summary"
+Description: "An overall summary of a course of radiotherapy. Whenever new contributions in the scope of the same treatment are delivered, this resource is updated. One course can involve multiple prescriptions. Modalities and techniques and detailed dose information is captured at the prescription delivery level. The status is changed to complete when the course has been fully delivered or changed to stopped if terminated."
+* insert RadiotherapyRS
+* code = LNC#68602-2 // Radiation oncology summary note
+* extension contains 
+    TreatmentIntent named treatmentIntent 0..1 MS and
+    TerminationReason named terminationReason 0..1 MS and
+    RadiotherapyPrescribedFractions named radiotherapyPrescribedFractions 0..1 MS and
+    RadiotherapyDeliveredFractions named radiotherapyDeliveredFractions 0..1 MS and
+    RadiotherapyTotalDosePlanned named radiotherapyTotalDosePlanned 0..1 MS and
+    RadiotherapyTotalDoseDelivered named radiotherapyTotalDoseDelivered 0..1 MS
 
-Conformance statement:
 
-Procedure resources associated with an mCODE patient with Procedure.category SNOMED-CT 53438000 MAY conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patient that would reasonably be expected to conform to this profile SHOULD be published in this form. Specifically, we expect that any radiation therapy related to the treatment of a `PrimaryCancerCondition` or `SecondaryCancerCondition` would be published in this form."
-* insert CancerRelatedProcedureCommonRuleSet
-// Do not insert the category slicing rules because Procedure.category is 0..1. 
-* category = SCT#53438000 //"Radiation therapy procedure or service (procedure)"
-* code from RadiationProcedureVS (extensible)
-* bodySite from RadiationTargetBodySiteVS (extensible)
-* extension contains TerminationReason named terminationReason 0..1 MS
+RuleSet:  RadiotherapyPrescriptionDeliveryRS
+* insert RadiotherapyRS
+// TO DO -- specify the MS elements
+* partOf only Reference(Procedure)
```

Comment:
Presumably this is `Reference(Procedure)` rather than `Reference(RadiotherapyCourseSummary )` because BrachytherapyPrescriptionDelivery doesn't use RadiotherapyCourseSummary. If this is the case, I think it should be documented somewhere (see suggestion on next line), and the rational should be explained.

Reply:
See above. This was an inconsistency...both PrescriptionDeliveries should reference a RadiotherapyCourse.